docs/modules/ROOT/pages/get-started.adoc

```diff
@@ -203,6 +203,8 @@ include::ROOT:example$/management-center.yaml[]
 
 WARNING: StatefulSet does not support updates to `volumeClaimTemplates` field, so `persistence` field should be set only at the creation of the custom resource. Any update to the `persistence` field will not affect the Management Center.
 
+NOTE: If you want to use an existing `PersistentVolumeClaim`, in the ManagementCenter CR you can set its name in the field `.spec.persistence.existingVolumeClaimName`. 
```

Comment:
A few questions:

- Is this for saving the `hazlecast-mc` directory?

- What's the default behavior? Where is the data persisted?

- Would a user be required to create and configure a `PersistentVolume` kind? And would that mean the `name` field actually refers to `metadata.name`?

Reply:
- It is for saving `/data` directory. If '.spec.persistence' is enabled, we create a `PersistentVolumeClaim` for users behalf.
- As default `.spec.persistence` is enabled, so we create the PVC for the user.
- If user's K8s cluster administrator or provider have configured a dynamic volume provisioner, they can use [StorageClasses](https://kubernetes.io/docs/concepts/storage/storage-classes/) to create `PersistentVolume`s automatically when they create `PersistentVolumeClaim`. We assume that user is doing what they are doing if they want to use an existing `PersistentVolumeClaim`.

Reply:
Thanks 😄  What's in `/data`? And where does our `PersistentVolumeClaim` store the data by default?

Reply:
Is this `/data` directory the same as the `hazelcast-mc` directory? The reason I ask is that it will be useful for K8 users to know why MC needs a `PersistentVolumeClaim` and where that data is stored by default. With this information, they can make an informed decision about whether to provide `.spec.persistence.existingVolumeClaimName`

Reply:
Yes, it is the mc-home directory. In the MC logs it can be seen as: `exec java --add-opens java.base/java.lang=ALL-UNNAMED -server -Dhazelcast.mc.home=/data`